### DISCUSSION TITLE
Using pushReplacement

### DISCUSSION THREAD

**Discussion by frankiechz** [csells/go_router#154] [Q&A]
Non-web question. Is there an example or any insight into how to use Navigator.pushReplacement (or any other method) to replace a current view on the stack? (I don't want me user to push the AppBar back button and go back.) Navigator.pushReplacement requires a Route object.

Thank you,
A new Flutter user

**Comment by csells** [comment]
You can replace the whole stack using `GoRouter.of(context).go(...)`. If you don't want the user to be able to go back, then going to a top level route will give you want you want.
